src/IdempotencyTokenMiddleware.php

```diff
@@ -52,11 +64,12 @@ public function __invoke(
             $operation = $this->service->getOperation($command->getName());
             $members = $operation->getInput()->getMembers();
             foreach ($members as $member => $value) {
-                if (!empty($value->toArray()['idempotencyToken'])) {
+                if (isset($value->toArray()['idempotencyToken'])) {
```

Comment:
I thought about using `isset` originally, then it could potentially allow an empty string as input. Thoughts?